lib/sqlalchemy/dialects/oracle/base.py

```diff
@@ -760,12 +760,18 @@ def visit_datetime(self, type_, **kw):
     def visit_float(self, type_, **kw):
         return self.visit_FLOAT(type_, **kw)
 
+    def visit_double(self, type_, **kw):
+        return self.visit_DOUBLE_PRECISION(type_, **kw)
+
     def visit_unicode(self, type_, **kw):
         if self.dialect._use_nchar_for_unicode:
             return self.visit_NVARCHAR2(type_, **kw)
         else:
             return self.visit_VARCHAR2(type_, **kw)
 
+    def visit_DOUBLE(self, type_, **kw):
```

Comment:
remove this, again folks would use "Double" if they want a generic "Double".

Reply:
Removed